apps/web/modules/event-types/views/event-types-single-view.tsx

```diff
@@ -338,7 +346,15 @@ const EventTypePage = (props: EventTypeSetupProps & { allActiveWorkflows?: Workf
       slotInterval: eventType.slotInterval,
       minimumBookingNotice: eventType.minimumBookingNotice,
       metadata,
-      hosts: eventType.hosts,
+      hosts: eventType.hosts.map((host) => {
+        const member = eventType.team?.members.find((mem) => mem.user.id === host.userId) || null;
+        return {
+          ...host,
+          avatar: member?.user?.avatarUrl,
+          label: member?.user?.name,
+          availability: null,
+        };
```

Comment:
This won't be needed along with other places where you are updating host with avatar and label

Reply:
Updated